### ISSUE TITLE
Display Junk messages as text only

### ISSUE THREAD

**Issue by Leryan** [RainLoop/rainloop-webmail#715]
Hi,

I inadvertently opened a message stored in my Junk folder, and its content was displayed. This is armful since junk messages can embed malicious code and/or fetch malicious extern resources.

Display these messages as full text would be a first barrier against that.

You may add a banner to let the user show it's original content.

**Comment by megadr01d** [comment]
:+1:

**Comment by CodeAlDente** [comment]
I agree, that would be useful. Too bad this has not been integrated yet.
